string-constants-doc/string-constants/translation-guide.scrbl

```diff
@@ -0,0 +1,161 @@
+#lang scribble/manual
+
+@title{How To Translate DrRacket's String Constants in Your Language}
+@author[(author+email "Alexander Shopov" "ash@kambanaria.org"  #:obfuscate? #t)]
+
+This is a short hands on guide how to translate the strings used
+within DrRacket's GUI. It is targeted at people willing to translate
+the interface who may not know all ins and outs of Racket or Scheme.
+
+@; ----------------------------------------------------------------------
+
+@section{Introduction}
+
+DrRacket is a GUI: editor and runner for the many languages you can
+use via Racket — the language developer’s language. Currently
+existing translations are English Danish, French, Finnish, German,
+Portuguese, Russian, Ukrainian, Bulgarian, Chinese (traditional and
+simplified) and Japanese. Dutch, Korean and Spanish have not been
+updated in a long while and they still need relicensing which makes
+them hidden in the interface.
```

Comment:
I think this will change over time so it is probably better to say "There are translations for more than a dozen languages, some of which are more actively maintained than others." or something more vague like that.

Reply:
Done